tool/python/examples/mnist_rbm1.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import sys, os
+sys.path.append(os.path.join(os.path.dirname(__file__),'..')) 
+from singa.model import *
+from singa.datasets import rbm 
+
+rbmid = 1
+X_train, X_test, workspace = rbm.load_data(rbmid)
+
+m = Sequential('rbm'+str(rbmid), sys.argv)
```

Comment:
instead of Sequential, Can I call/create "Bidirectional", inherited from Model class?